lua/astrocommunity/completion/avante-nvim/init.lua

```diff
@@ -0,0 +1,42 @@
+return {
+  "yetone/avante.nvim",
+  event = "VeryLazy",
+  lazy = false,
```

Comment:
can be removed if command and keybindings are added

Reply:
removed in the latest commit